optimize/self-managed/optimize-deployment/migration-update/camunda-8/3.10-to-3.11_8.3.md

```diff
@@ -47,7 +41,7 @@ With this release, the minimum version of Java that Optimize supports is now Jav
 
 ### Plugins
 
-Optimize now runs with Spring Boot 3. As a result, some plugin interfaces have been updated accordingly. More specifically, the [Engine Rest Filter Plugin](./../plugins/engine-rest-filter-plugin.md) and the [Single-Sign-On Plugin](./../plugins/single-sign-on.md) now import jakarta dependencies. If you use these plugins and are updating from version 3.10.3 or earlier, you will need to adjust your implementation accordingly.
+Optimize now runs with Spring Boot 3. As a result, some plugin interfaces have been updated accordingly. More specifically, the [Engine Rest Filter Plugin](./../../plugins/engine-rest-filter-plugin.md) and the [Single-Sign-On Plugin](./../../plugins/single-sign-on.md) now import jakarta dependencies. If you use these plugins and are updating from version 3.10.3 or earlier, you will need to adjust your implementation accordingly.
```

Comment:
I know this is not new but we can consider this an opportunity to clean things up. Plugins were never supported in C8, so let's just remove them here